includes/namespace.php

```diff
@@ -38,17 +38,18 @@
  * @return void
  */
 function setup_new_site() {
-	$story = new Story_Post_Type( new Experiments(), new Meta_Boxes() );
-	$story->init();
+	$story = new Story_Post_Type( new Experiments(), new Meta_Boxes(), new Decoder() );
+	$story->register();
```

Comment:
Could be (for a later PR, as discussed):
```php
Services::get( 'story_post_type' )->register()
```

Reply:
Let's handle this here https://github.com/google/web-stories-wp/issues/5062